src/node/Launcher.ts

```diff
@@ -203,6 +203,7 @@ class ChromeLauncher implements ProductLauncher {
 
   defaultArgs(options: BrowserLaunchArgumentOptions = {}): string[] {
     const chromeArguments = [
+      '--allow-pre-commit-input', // TODO(crbug.com/1320996): neither headful nor headless should rely on this flag.
```

Comment:
Can we add this argument only for the related tests? (or is it needed in all tests?)

Reply:
It's also needed for runtime (not only tests) in headful unfortunately

Reply:
Oh understood, thank you!